src/dso_api/dynamic_api/filterset.py

```diff
@@ -51,15 +51,68 @@ def filterset_factory(model: Type[DynamicModel]) -> Type[DynamicFilterSet]:
     # Determine which fields are included:
     # Excluding geometry fields for now, as the default filter only performs exact matches.
     # This isn't useful for polygon fields, and excluding it avoids support issues later.
-    fields = {f.attname: _get_field_lookups(f) for f in model._meta.get_fields()}
+    fields = {
+        f.attname: _get_field_lookups(f) for f in model._meta.get_fields()
+        if isinstance(f, (models.fields.Field, models.fields.related.ForeignKey))
+    }
+
+    extra_fields, filters = generate_relation_filters(model)
 
     # Generate the class
     meta_attrs = {
         "model": model,
         "fields": fields,
     }
     meta = type("Meta", (), meta_attrs)
-    return type(f"{model.__name__}FilterSet", (DynamicFilterSet,), {"Meta": meta})
+    return type(
+        f"{model.__name__}FilterSet", (DynamicFilterSet,), {"Meta": meta, **filters}
+    )
+
+
+def generate_relation_filters(model: Type[DynamicModel]):
+    """
+    Generates additional filters for relations, including sub items.
+    """
+    fields = dict()
+    filters = dict()
+
+    for relation in model._meta.related_objects:
+        schema_fields = dict([(f.name, f) for f in model._table_schema.fields])
+        if relation.name not in schema_fields:
+            fields[relation.name] = ["exact"]
+            continue
+        if not schema_fields[relation.name].is_nested_table:
+            continue
+
+        relation_properties = schema_fields[relation.name]['items']['properties']
+        for field_name, field_schema in relation_properties.items():
+            # contert space separated property name into snake_case name
+            model_field_name = format_field_name(field_name)
+            model_field = getattr(relation.related_model, model_field_name).field
+            filter_class = DSOFilterSet.FILTER_DEFAULTS.get(model_field.__class__)
+            if filter_class is None:
+                # No mapping found for this model field, skip it.
+                continue
+            filter_class = filter_class["filter_class"]
+            # Filter name presented in API
+            filter_name = "__".join([
```

Comment:
Moest dit vanwege de haalcentraal API? Of hebben we dat zelf bedacht. Zou het niet fraaier zijn om relaties met een `.` te scheiden?